ncard_app/templates/tables/events.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'events/base.html' %}
+{% load render_table from django_tables2 %}
+{% load bootstrap5 %}
+{% load crispy_forms_tags %}
+
+{% block title %}Events | NCARD Table{% endblock %}
+
+{% block content%}
+<div class="col-md-8">
+            <h1 class="ncard-dark-green">Events</h1>
+</div>
+
+<div class="row mb-4">
+
+    {% if filter %}
+        <div class="col-md-2">
+            <form action="" method="get" class="form form-inline">
+                {% bootstrap_form filter.form layout='inline'  %}
+{#            <button type="submit" class="ncard_CTA">Search</button>#}
```

Comment:
This line is commented out, just remove it

Reply:
fixed it already